spec/std/array_spec.cr

```diff
@@ -279,7 +279,15 @@ describe "Array" do
 
     it "gets with start and count" do
       [1, 2, 3][1, 3]?.should eq([2, 3])
-      [1, 2, 3][4, 0]?.should be_nil
+    end
```

Comment:
You shouldn't remove this expectation.

Reply:
Done.

Reply:
Think I removed that accidentally. Sorry about that.